search/api.py

```diff
@@ -74,13 +84,30 @@ def perform_search(
         log_search_params=log_search_params,
     )
 
+    processing_time = {
+        "start": time.time()
+    }
+
     # post-process the result
     for result in results["results"]:
         result["data"] = SearchResultProcessor.process_result(result["data"], search_term, user)
 
     results["access_denied_count"] = len([r for r in results["results"] if r["data"] is None])
     results["results"] = [r for r in results["results"] if r["data"] is not None]
 
+    processing_time['end'] = time.time()
+    search_time['end'] = time.time()
+
+    processing_time_in_seconds = processing_time['end'] - processing_time['start']
+    search_time_in_seconds = search_time['end'] - search_time['start']
+    filter_generation_time_in_seconds = filter_generation_time['end'] - filter_generation_time['start']
+
+    track.emit("edx.course.search.executed", {
+        "search_term": search_term,
+        "processing_time_in_seconds": processing_time_in_seconds,
```

Comment:
I think we should emit not only the time in seconds, but also we should emit the start time and end time of this process.